src/components/Header.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+
+export function Header(): JSX.Element {
+  return (
+    <header className="p-3 mb-3 align-right" style={{ background: '#009cab' }}>
+      <div className="container">
+        <HeaderLink to="/">
+          <HeaderLogo />
+        </HeaderLink>
+
+        <nav className="float-right">
```

Comment:
I personally would probably use flexbox or this. e.g. `ml-auto` on this, and `flex` on the container. I think there is a container-flex thing in bootstrap, but you should check.

Reply:
Okay, I'll leave this as an improvement issue for other people to work on.

Reply:
https://github.com/exercism/v3-dashboard/issues/92